docs/modules/secret-operator/pages/security.adoc

```diff
@@ -0,0 +1,16 @@
+= Security
+
+== Container Privileges
```

Comment:
Not sure if this one is correct though
```suggestion
== Container privileges
```

Reply:
lowercase is correct (sentence casing for titles)